apps/rehash.c

```diff
@@ -366,9 +366,12 @@ static int do_dir(const char *dirname, enum Hash h)
         BIO_printf(bio_err, "Skipping %s, can't write\n", dirname);
         return 1;
     }
-    buflen = strlen(dirname);
-    pathsep = (buflen && !ends_with_dirsep(dirname)) ? "/": "";
-    buflen += NAME_MAX + 1 + 1;
+    dirlen = strlen(dirname);
+    if (dirlen && !ends_with_dirsep(dirname)) {
```

Comment:
nit: `dirlen != 0`

Reply:
I just moved existing code, but sure, I'll change it.

Reply:
done.